### ISSUE TITLE
ERROR - cannot find Chrome (new location for newer versions on Windows) [fixed]

### ISSUE THREAD

**Issue by haidvams** [tebelorg/RPA-Python#224]
[RPA][ERROR] - following happens when starting TagUI...

ERROR - cannot find Chrome at "C:\Program Files (x86)\Google\Chrome\Application\chrome.exe"
ERROR - or at user folder "C:\Users\koios\AppData\Local\Google\Chrome\Application\chrome.exe"
update chrome_command setting in tagui\src\tagui.cmd to your chrome.exe

**Comment by kensoh** [comment]
Hi @haidvams this is happening because newer Chrome / Windows version seem to install Chrome in a new different location. This will be fixed in a future release, for the time being you can resolve it by doing the following.

1. Get the location of where TagUI is installed with
```py
import os; print(os.environ['APPDATA'])
```

2. There will be a `tagui` folder there. Inside there, edit the file src\tagui.cmd and change below line
```
set chrome_command=C:\Program Files (x86)\Google\Chrome\Application\chrome.exe
```

3. To below (or wherever Chrome is installed on your computer and chrome.exe is found)
```
set chrome_command=C:\Program Files\Google\Chrome\Application\chrome.exe
```

**Comment by haidvams** [comment]
thank you , I got

**Comment by shridharampra** [comment]
Thanks. Had started with RPA yday night and got into this issue. This helped.

**Comment by kensoh** [comment]
Implemented in v1.29, available with `pip install rpa --upgrade`
